droit_glg.csl

```diff
@@ -0,0 +1,816 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" et-al-min="4" et-al-use-first="4" et-al-subsequent-min="4" et-al-subsequent-use-first="4" page-range-format="expanded" demote-non-dropping-particle="sort-only" default-locale="fr-FR">
+  <info>
+    <title>Droit_GLG</title>
+    <title-short>Droit_GLG</title-short>
+    <id>http://www.zotero.org/styles/droit_glg</id>
+    <link href="http://www.zotero.org/styles/droit_glg" rel="self"/>
+    <link href="http://www.zotero.org/styles/ens-de-lyon-centre-d-ingenierie-documentaire" rel="template"/>
+    <link href="http://cid.ens-lyon.fr/articles/zotero_style_cid.pdf" rel="documentation"/>
+    <link href="https://drive.google.com/file/d/1yROMcnNg5QjW_YkzxcMdp-TsJHJQFAHs/view?usp=sharing" rel="documentation"/>
+    <author>
+      <name>Grégoire Lemarchand-Ghica</name>
+      <email>Grégoire.lemarchand-ghica@u-paris2.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="philosophy"/>
+    <category field="humanities"/>
+    <category field="political_science"/>
+    <category field="law"/>
+    <category field="droit"/>
+    <summary>Style modifié depuis ENS de Lyon (Stéphane Marchand et Anatole Lucet)</summary>
+    <published>2025_03_01</published>
+    <updated>2025-03-03T10:27:03+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="fr">
+    <terms>
+      <term name="cited">op. cit.</term>
+      <term name="page" form="short">p.</term>
+      <term name="editor" form="short">éd.</term>
+      <term name="editorial-director" form="short">dir.</term>
+      <term name="translator" form="short">trad.</term>
+      <term name="prefacier" form="short">préf.</term>
```

Comment:
prefacier is not a name that exists in the Citation style language and will throw an error.